### ISSUE TITLE
Access denied to Windows nightly build

### ISSUE THREAD

**Issue by ghost** [chrippa/livestreamer#1554]
http://livestreamer-builds.s3.amazonaws.com/livestreamer-latest-win32.zip link doesn't work any more.

**Comment by xtj-9182** [comment]
@voltagex 
yeah livestreamer has been abandoned. I suggest you check out [streamlink](https://github.com/streamlink/streamlink) it's a fork of livestreamer and is in active development currently.

**Comment by manpages** [comment]
@xtj-9182 thank you very much! Wouldn't hurt to put such information in README / on the documentation web-site :3

**Comment by xtj-9182** [comment]
@manpages 
correct me if I'm wrong but I think this isn't possible cause no one has permission to do so except @chrippa who has been gone for almost 2 years.

**Comment by manpages** [comment]
@xtj-9182 right. He clearly is alive (thank G-d), so I'll try reaching out to him in case he ignores highlight in this issue.
